docs/src/main/paradox/slick.md

```diff
@@ -0,0 +1,139 @@
+
+The Slick connector supports all the various ways Slick allows you to configure your JDBC database drivers, connection pools, etc., but we strongly recommend using the so-called ["DatabaseConfig"](http://slick.lightbend.com/doc/3.2.1/database.html#databaseconfig) method of configuration, which is the only method explicitly tested to work with the Slick connector.
+
+Below are a few configuration examples for other databases. The Slick connector supports all [databases supported by Slick](http://slick.lightbend.com/doc/3.2.1/supported-databases.html) (as of Slick 3.2.x)
+
+Postgres
+: @@snip (../../../../slick/src/test/resources/application.conf) { #config-postgres }
+
+MySQL
+: @@snip (../../../../slick/src/test/resources/application.conf) { #config-mysql }
+
+DB2
+: @@snip (../../../../slick/src/test/resources/application.conf) { #config-db2 }
+
+Oracle
+: @@snip (../../../../slick/src/test/resources/application.conf) { #config-oracle }
+
+SQL Server
+: @@snip (../../../../slick/src/test/resources/application.conf) { #config-sqlserver }
+
+Of course these are just examples. Please visit the [Slick documentation for `DatabaseConfig.fromConfig`](http://slick.lightbend.com/doc/3.2.1/api/index.html#slick.jdbc.JdbcBackend$DatabaseFactoryDef@forConfig(String,Config,Driver,ClassLoader):Database) for the full list of things to configure.
+
+### Closing a Database Session
+Slick requires you to eventually close your database session to free up connection pool resources. You would usually do this when terminating the `ActorSystem`, by registering a termination handler like this:
+
+Scala
+: @@snip (../../../../slick/src/test/scala/akka/stream/alpakka/slick/scaladsl/SlickSpec.scala) { #close-session }
+
+Java
+: @@snip (../../../../slick/src/test/java/akka/stream/alpakka/slick/javadsl/SlickTest.java) { #close-session }
+
+### Using a Slick Source
+The Slick connector allows you to perform a SQL query and expose the resulting stream of results as an Akka Streams `Source[T]`. Where `T` is any type that can be constructed using a database row.
+
+#### Plain SQL queries
+Both the Scala and Java DSLs support the use of [plain SQL queries](http://slick.lightbend.com/doc/3.2.1/concepts.html#plain-sql-statements).
+
+The Scala DSL expects you to use the special `sql"..."`, `sqlu"..."`, and `sqlt"..."` [String interpolators provided by Slick](http://slick.lightbend.com/doc/3.2.1/sql.html#string-interpolation) to construct queries.
+
+Unfortunately, the Java DSL does not support string interpolation because the underlying Slick project does not directly support Java. This means query strings will need to be manually prepared using plain Strings. Please watch out for SQL injection attacks while doing so!
+
+The following examples put it all together to perform a simple streaming query. The full source code for these examples can be found together with the unit tests of the Slick connector [on Github](https://github.com/akka/alpakka/tree/master/slick/src/test).
```

Comment:
Again, is it policy not to show the imports?

Reply:
Don't show await in example.

Reply:
Both fixed